src/main/groovy/be/cytomine/software/consumer/threads/ProcessingServerThread.groovy

```diff
@@ -43,18 +43,16 @@ class ProcessingServerThread implements Runnable {
         try {
             log.info("${this.processingServer.getStr("processingMethodName")}")
             processingMethod = AbstractProcessingMethod.newInstance(this.processingServer.getStr("processingMethodName"))
-            processingMethod.communication = new SSH(
-                    host: processingServer.getStr("host"),
-                    port: processingServer.getStr("port") as Integer,
-                    user: processingServer.getStr("username"),
-                    keyFilePath: Main.configFile.cytomine.software.sshKeysFile
-            )
+            String keyFilePath=Main.configFile.cytomine.software.sshKeysFile
+            def keyPathToReturn = """${keyFilePath}/${processingServer.getStr("host")}/${processingServer.getStr("host")}"""
```

Comment:
keyPath or something like this is not ok ? Why "ToReturn", you return nothing here